.github/workflows/gajira.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       if: ${{ !contains(github.event.pull_request.labels, 'snyk') }}
       run: |
         echo "Pull request not created by Snyk"
-        exit 0
+        exit 1
```

Comment:
If this step raises an exit code, and every new PR starts this job, then any PR's opened without Synk in the title will be marked as a failure.

This will result in a long list of failed pipeline runs, is this desired?